packages/react/src/FAQ/FAQ.tsx

```diff
@@ -85,12 +85,13 @@ const FAQHeading = forwardRef<HTMLHeadingElement, FAQHeadingProps>(
 
 type FAQSubheadingProps = BaseProps<HTMLHeadingElement> & {
   align?: 'start' | 'center'
+  as?: 'h2' | 'h3' | 'h4' | 'h5' | 'h6'
   children: string
 }
 
-function FAQSubheading({children, className, ...rest}: FAQSubheadingProps) {
+function FAQSubheading({children, className, as = 'h3', ...rest}: FAQSubheadingProps) {
   return (
-    <Heading as="h3" className={clsx(styles.FAQ__subheading, className)} {...rest}>
+    <Heading as={as} className={clsx(styles.FAQ__subheading, className)} {...rest}>
```

Comment:
I made a change where `FAQ.Subheading` can be adjusted as well. This was mainly so that if someone were to adjust `FAQ.Question` and they were also utilizing `FAQ.Subheading`, they'd be able to change both component's heading levels.

Reply:
thanks Tyler, that makes sense 👍. Worth also doing FAQ.Heading also for consistency?

Reply:
Definitely! I added `as` to `FAQ.Heading`, and kept the `size` prop since I see we utilize this for visual sizing too. Let me know if this wasn't the right approach 🤔.